README.md

```diff
@@ -12,7 +12,8 @@ The acmOpportunities Discord Bot is a tool designed for students in our club. Th
 1. Install [Python](https://www.python.org/downloads/).
 2. Pip install the dependencies of main.py using `pip install -r requirements.txt`.
 3. Set the required environment variables.
-4. To run the program manually, write `python ./main.py --days-needed 2`.
+4. Call the function `create()` on `line 277` in `main.py` **only once**.
```

Comment:
This number might change. I’d suggest not using the exact line number.

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/41e95dfe43d3122a24bffb772f72ecce05315faf)

Reply:
Are you saying to remove `Call the function `create()` on `line 277` in `main.py` **only once**`?

Reply:
> Are you saying to remove `` Call the function `create()` on `line 277` in `main.py` **only once** ``?

Just the `line 277` part.